app/uk/gov/hmrc/trackingconsentfrontend/assets/src/interfaces/preferenceCommunicatorFactory.ts

```diff
@@ -0,0 +1,18 @@
+const preferenceCommunicatorFactory = (window) => ({
+  sendPreferences: (userPreferences) => {
+    window.dataLayer = window.dataLayer || []
+    const preferences = userPreferences.getPreferences() || {}
+    console.log(preferences)
```

Comment:
Console.log in code - surprised linter hasn't complained.

Reply:
Doh, it would be good for the linter to complain over that.